exe/main.cc

```diff
@@ -182,8 +182,9 @@ int main(int argc, char *argv[]) {
     // Set up logging
 
     try {
-        Configuration::Instance().setupLogging();
-        LOG_DEBUG("pxp-agent logging has been initialized");
+        std::string version, loglevel;
+        std::tie(version, loglevel) = Configuration::Instance().setupLogging();
+        LOG_INFO("pxp-agent {1} started at {2} level", version, loglevel);
```

Comment:
I'd suggest returning only the log level from the `Configuration::Instance().setupLogging()` method and using the `PXP_AGENT_VERSION` here directly. The rationale being that the `PXP_AGENT_VERSION` define is relevant to all of the pxp-agent code not just logging, so it seems inappropriate to return its value from the logging setup function. Moreover by simplifying the `setupLogging()`'s return value we'll be able to avoid having to pack/unpack the two values to/from the `std::pair`.